lisa/tools/python.py

```diff
@@ -49,17 +51,31 @@ def _install(self) -> bool:
         self.node.os.install_packages(package_name)
         return self._check_exists()
 
-    def install_packages(self, packages_name: str) -> None:
-        cmd_result = self.run(
-            f"install -q {packages_name}",
-        )
+    def install_packages(self, packages_name: str, install_path: str = "") -> None:
+        node = self.node
+        cmd_line = f"install -q {packages_name}"
+
+        envs = {}
+        if install_path != "":
+            tagert_path = install_path + "/python_packages"
+            node.tools[Mkdir].create_directory(tagert_path, sudo=True)
+            cache_path = install_path + "/tmp"
+            node.tools[Mkdir].create_directory(cache_path, sudo=True)
+            node.tools[Chmod].update_folder(tagert_path, "777", sudo=True)
```

Comment:
why use chmod one by one? The command use recursive already, the install_path should be applied for both sub directories.

Reply:
In some Ubuntu env, the find_partition_with_freespace() will return root "/" folder for "install_path". If we run "chmod install_path", it will change mode for everything in the root folder, which is not correct.